src/launcher.cr

```diff
@@ -0,0 +1,14 @@
+require "raven"
+require "logger"
+require "pg"
+require "pg/pg_ext/big_decimal"
+require "discordcr"
+require "big"
+require "big/json"
+
+require "./discordtipbot"
+require "./discordtipbot/*"
+
+unless ENV["TIPBOT_ENV"]? == "test"
```

Comment:
This was only necessary because we had one single "main" file, and when the spec suite loaded the code base, it would try to run the bot.

This file can simply be:

```cr
require "./discordtipbot"
DiscordTipBot.new
```

and the spec suite wouldn't require this file (`launcher.cr`), only `discordtipbot.cr`, accomplishing the same thing without the need for an `ENV` var. 

If you remove this, be sure to remove it from `travis.yml` as well

https://github.com/z64/hornet/blob/rewrite/src/entrypoint.cr